src/backend/distributed/commands/table.c

```diff
@@ -574,24 +579,53 @@ PreprocessAlterTableStmt(Node *node, const char *alterTableCommand,
 		ErrorIfUnsupportedAlterTableStmt(alterTableStatement);
 	}
 
+	if (ShouldSyncTableMetadata(leftRelationId))
+	{
+		EnsureCoordinator();
+	}
```

Comment:
What is this for? Does this affect anything else?

Reply:
Shouldn't that be the case since in postprocess if MX we send commands to worker nodes?